test/travis.yml

```diff
@@ -45,6 +45,13 @@ install:
   - sudo apt-get install pigz
   - if [ -n "$INSTALL_PV" ]; then sudo apt-get install pv; fi
   - sudo pip install sphinx sphinx-rtd-theme
+# Ubuntu Trusty symlinks /dev/shm -> /run/shm; the latter is unavailable after
+# entering the namespaces and we get ENOENT. Work around this bug. See:
+# https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=851427
+# https://wiki.debian.org/ReleaseGoals/RunDirectory
+  - sudo rm /dev/shm
+  - sudo mkdir /dev/shm
+  - sudo mount --bind /run/shm /dev/shm
 
```

Comment:
In case they change this behavior you might want to protect it with ```if test -h /dev/shm```.

Reply:
That's a good point. In this particular case, we're targeting a specific environment in Travis (Ubuntu 14.04 IIRC). Newer versions of Ubuntu don't have this bug. So the Travis tests will break then and we'll remove the above workaround.